packages/origin-247-transfer/src/utils/parseRedisUrl.ts

```diff
@@ -0,0 +1,57 @@
+import { defaults } from 'lodash';
+import { parse as urllibParse } from 'url';
+
+/**
+ * We support only REDIS_URL env variable,
+ * and we want to use io-redis cache manager,
+ * therefore https://github.com/dabroek/node-cache-manager-ioredis/pull/1
+ * and https://github.com/luin/ioredis/blob/4680211fe853831f9ff3a3eb69f16d5db6bfbabd/lib/utils/index.ts
+ */
+
+function isInt(value): value is string {
+    const x = parseFloat(value);
+    return !isNaN(value) && (x | 0) === x;
+}
+
+export function parseRedisUrl(url) {
```

Comment:
it's not my function, see comment

Reply:
It's pasted from here? https://github.com/luin/ioredis/blob/4680211fe853831f9ff3a3eb69f16d5db6bfbabd/lib/utils/index.ts#L243-L292

Could you then put this link in the comment above just to give attributions? Thanks.

Reply:
It's a link to the file, not the exact function. And it's not above that function but the whole file + it doesn't explicitly say it's copied.

Reply:
Okey, gonna fix that